### PR TITLE
added redis conf file

### PR DESCRIPTION
**Pull request by tilast**

### CODE REVIEW

File: packages/redis.rb [hunk 1]
```diff
@@ -12,3 +13,18 @@
...
   end
 end
+
+package :update_redis_conf do
+  config_file_path = "/etc/redis/redis.conf"
+  config_template = File.join(File.dirname(__FILE__), 'configs', 'redis.conf')
+
+  file config_file_path, contents: File.read(config_template), sudo: true
+
+  runner "chmod 755 #{config_file_path}"
```
Comment (ololyay) [inline packages/redis.rb]: i'm not sure about this line

**Comment by ololyay** [comment]
maybe its better to add `bind` option to config on `post :install` instead of writing full config

**Comment by rilian** [comment]
can you plz remove all the noise

**Comment by ololyay** [comment]
lgtm

**Comment by rilian** [comment]
looks good to me

**Pull request merged**